src/index.ts

```diff
@@ -64,35 +67,81 @@ function serviceUnavailable(message: string): APIGatewayProxyResult {
   };
 }
 
+const putConsentToFirehose =
+    (cmpCookie: CMPCookie, callback: APIGatewayProxyCallback,
+     streamName: string) => {
+      fh.putRecord(
+          {
+            DeliveryStreamName: streamName,
+            Record: {Data: new Buffer(JSON.stringify(cmpCookie))}
+          },
+          (err, data) => {
+            if (err) {
+              console.log(err, err.stack);
+            }  // an error occurred
+            else {
+              console.log(data);
+            }  // successful response, remove later.
+            callback(err, ok(data.RecordId));
+          });
+    };
+
+const handleAmp =
+    (event: APIGatewayProxyEvent, context: Context,
```

Comment:
I thought we were using ALBs for this? Do they use the same data structures for invocation?

Reply:
(obviously it works, so I assume this is fine!)